### ISSUE TITLE
'Borgo (And possibly AI) AI IM is unable to send messages

### ISSUE THREAD

**Issue by Chada-1** [Aurorastation/Aurora.3#8956]
Just as I said, it's as simple as the 'Send Message' button not computing.

The send message button must compute, I'm not sure how to fix it, but it's v. necessary to fix a v. long standing problem with 'borgos and I'd v. much appreciate it if anyone who does know how would take a look.

**Comment by aleksix** [comment]
Just to clarify - the issue here is that disabled borgs can't send PDA messages or normal borgs not being able to send PDA messages? The PR with the issue mentions disabled ones.
For the disabled borgs, this seems to be by design, so I'm not sure this is an issue.
For the working borgs - there are a few bugs, but the worst is that you can only reply by clicking on the name - you can not send messages to borgs, but their messages go through just fine.

**Comment by Chada-1** [comment]
It wasn't always this way. If it is by design, it's awful design, since literally all it leads to is a borg player being stuck in an area unable to do anything for like 30+ minutes.

The PR mentions disabled because I went code diving to fix the issue, and found it's been ported to a brand new system. When I tested it ingame, it didn't matter if you were disabled or not,you couldn't send a PDA message.
